index.html

```diff
@@ -56,5 +56,190 @@ <h1 class="Intro">
+            sign-ups required.
+          </p>
+          <ul class="skill--list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="see--project">
+           See project
+          </button>
+        </div>
+      </div>
+
+      <div class="parent--container">
+        <div class="first--child">
+          <a href="#"><img src="images/3rd pic.svg" alt="" /></a>
+        </div>
+
+        <div class="second--child">
+          <h2>Tonic</h2>
+          <ul class="ul--list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill--list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="see--project">
+            See project
+          </button>
+        </div>
+      </div>
+
+      <div class="parent--container">
+        <div class="first--child">
+          <a href="#"><img src="images/4th mobile pic.svg" alt="" /></a>
+        </div>
+
+        <div class="second--container">
+          <h2>Multi-Post Stories</h2>
+          <ul class="ul--list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill--list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="see--project">
+            See project
+          </button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about--section">
+      <div>
+        <h2>About Myself</h2>
+        <p class="about--text">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <p class="hero-connect-text">LET'S CONNECT</p>
+        <ul class="hero-icon-container">
+          <li>
+            <a href="https://twitter.com/AyokunnumiA">
+              <img src="images/Vector.svg" alt="" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/ayokunnumiomololu">
+              <img src="images/Linkedin icon.svg" alt="" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/ayokunnumiomololu">
+              <img src="images/Vector (1).svg" alt="" />
+            </a>
+          </li>
```

Comment:
[OPTIONAL] This is an optional requirement but recommended so kindly don't skip this requirement there are some spacing problems with your `about section` kindly fix it :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face:

| Your Design | Zaplin Design|
|--------|--------|
| ![Screenshot from 2023-06-03 23-32-00](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/45226848/ba5a03bb-7fa3-4440-b97b-488bb93e7d75) | ![Screenshot from 2023-06-03 23-32-26](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/45226848/e72271f4-8ad6-46b7-ae93-7177de17e9b4) |